docs/en/multi_modal/deepseek_vl2.md

```diff
@@ -0,0 +1,42 @@
+# DeepSeek-VL2
```

Comment:
may need to include this doc in `index.rst` https://github.com/InternLM/lmdeploy/blob/main/docs/en/multi_modal/index.rst